### DISCUSSION TITLE
Receipt API empty shipping_method in Etsy V3

### DISCUSSION THREAD

**Discussion by itsdarrylnorris** [etsy/open-api#881] [Q&A] [answered]
I am working on a migration between Etsy v2 to Etsy V3. The endpoints that I am working on are: 

- Etsy V2 endpoint: `get /shops/{shop_name}/receipts?includes=Transactions/MainImage,Country`
- Etsy V3 endpoint: `get /v3/application/shops/{shop_id}/receipts`

In Etsy V2, there were an `shipping_details.shipping_method` and it looks like the move it to `transactions[].shipping_method`. However, when I am querying the V3 API `transactions[].shipping_method` is equal to null and in V2 `shipping_details.shipping_method` has the correct value. I am looking in the correct place? Any idea why `transactions[].shipping_method` is null?

Do you guys know if possible to get these values by calling a different API?

**Comment by Gareth064** [comment, marked as answer]
When this field is Null, this means that the buyer is using the default shipping profile on the listing aka "Standard Shipping"

It will contain a value if the buyer chooses a shipping upgrade.

Reference: https://github.com/etsy/open-api/issues/271#issuecomment-956764696

**Reply by itsdarrylnorris** [reply]
Great thanks for letting me know

**Comment by fannigurt** [comment]
Not working for my case, added answer here https://github.com/etsy/open-api/discussions/956#discussioncomment-5625337
